iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -425,6 +425,15 @@ public void commitAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable
         case SETPARTITIONSPEC:
           IcebergTableUtil.updateSpec(conf, icebergTable);
           break;
+        case ROLLBACK:
+          LOG.info("Executing rollback operation on iceberg table. If you would like to revert rollback you could " +
```

Comment:
If I understand correctly a `rollback` operation just adds a new snapshot at the end of the history. So  if we want to roll back a rollback, then we just have to do another rollback.

If the above is true, then I think it would be easier for the user to understand

Reply:
Yes and no :)
Iceberg rollback creates a new snapshot, but it also keeps the historical evolution. It is easier to understand through an example. 
Lets have the following snapshots:
A -> B -> C -> D

After a rollback to let's say B, the snapshot chain will look like
A -> E
where E will be a new snapshot, but all it's properties (except for the ID) will match B snapshot's properties.